### DISCUSSION TITLE
Can no longer collapse directories in File tree view

### DISCUSSION THREAD

**Discussion by heaths** [community/community#113359] [Pull Requests]
### Select Topic Area

Bug

### Body

In the last couple of days, I can no longer collapse a directory in a PR's File tree view e.g.,

![image](https://github.com/community/community/assets/1532486/a4ca77d3-63c5-49fe-a427-0e7ac1cc6091)

When I click on the up/down icon or the directory itself, nothing happens. This is a regression since it worked prior since I first opted into the tree view beta.

**Comment by chrisakinrinade** [comment]
Exact same issue for me. Also started a couple days ago.

**Comment by mfurkanayhan** [comment]
Despite trying different browsers, ensuring JavaScript is enabled, and clearing cache and cookies, the issue persists for me as well. This issue occurs exclusively within commits; I have not encountered it while navigating through the repository itself.
![bug](https://github.com/community/community/assets/75537063/5a63e6e5-8723-4caf-82ff-a6e4c29ad859)

**Reply by mfurkanayhan** [reply]
Additionally, when I click on a file, it does not remain highlighted as selected after the click.

**Reply by IOrlandoni** [reply]
![62c08f75d8b792cc857a5ab4396788bb1-ezgif com-crop](https://github.com/orgs/community/discussions/113359)

**Reply by mfurkanayhan** [reply]
https://github.com/orgs/community/discussions/113359#discussioncomment-8892502

**Comment by harsh-leen** [comment]
Same issue

**Comment by mfurkanayhan** [comment]
Problem solved

**Reply by hornworm61** [reply]
How was the problem solved?

**Comment by Alkhatri77** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Body
> 
> In the last couple of days, I can no longer collapse a directory in a PR's File tree view e.g.,
> 
> ![image](https://github.com/community/community/assets/1532486/a4ca77d3-63c5-49fe-a427-0e7ac1cc6091)
> 
> When I click on the up/down icon or the directory itself, nothing happens. This is a regression since it worked prior since I first opted into the tree view beta.
